config/samples/agent-install.openshift.io_v1beta1_agentserviceconfig.yaml

```diff
@@ -30,3 +30,7 @@ spec:
     rootFSUrl: https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.8/4.8.2/rhcos-live-rootfs.x86_64.img
     url: https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.8/4.8.2/rhcos-4.8.2-x86_64-live.x86_64.iso
     version: 48.84.202107202156-0
+  - openshiftVersion: '4.9'
```

Comment:
Do we need to check support for 4.9 in the operator?
As we remove old versions on OPERATOR_OPENSHIFT_VERSIONS: https://github.com/openshift/assisted-service/blob/master/hack/generate.sh#L72
But we use only 4.8 on deploy: https://github.com/openshift/assisted-service/blob/master/deploy/operator/common.sh#L36